static/js/server_events_dispatch.js

```diff
@@ -678,7 +678,7 @@ export function dispatch_normal_event(event) {
                 $("#go-to-default-view-hotkey-help").toggleClass("notdisplayed", !event.value);
                 break;
```

Comment:
We need to remove this `break` statement to correctly update `Escape key navigates to default view` setting.